### PR TITLE
Standardize naming/style/grammar in READMEs and comments

### PR DESCRIPTION
**Pull request by JennyTee**
Came across some instances where we were saying "agent" instead of "the GX Agent". Updated these and also updated some style/grammar along the way to better adhere to the styleguide

### CODE REVIEW

**Review by anthonyburdi — APPROVED** [review]
Thank you for splitting this out! LGTM!

File: examples/agent/cloudformation/README.md [hunk 1]
```diff
@@ -28,17 +28,17 @@
...
 
-5. Create the cloudformation stack
+5. Create the CloudFormation stack
 
 ```
 aws cloudformation create-stack --stack-name gx-agent --template-url https://gx-agent-cloudformation.s3.amazonaws.com/agent-stack.yaml --parameters file://parameters.json --capabilities CAPABILITY_IAM
 ```
 
-6. Go into the AWS console under the CloudFormation page to see that the gx-agent has been spun up. Then go to ECS to see your task running, in the logs you should see that the agent has spun up.
+1. Go into the AWS Management Console under the CloudFormation page to see that the GX Agent has been spun up. Then go to ECS to see your task running--in the logs you should see that the GX Agent has been spun up.
```
Comment (JennyTee) [inline examples/agent/cloudformation/README.md:37]: aw the numbers got messed up...will update

Comment (JennyTee) [reply examples/agent/cloudformation/README.md]: fixed

**Review by Shinnnyshinshin — APPROVED** [review]
This is great. Thank you @JennyTee!

**Pull request merged**